src/addrdb.cpp

```diff
@@ -23,7 +23,7 @@ bool SerializeDB(Stream& stream, const Data& data)
 {
     // Write and commit header, data
     try {
-        CHashWriter hasher(SER_DISK, CLIENT_VERSION);
+        CHashWriter hasher(SER_DISK, stream.GetVersion());
```

Comment:
501de309e682f2edddb7e51e55665e034e1a86be, while this line is touched, it seems natural to inherit the stream type as well:

```suggestion
        CHashWriter hasher(stream.GetType(), stream.GetVersion());
```

Reply:
Done (ages ago).